src/http/response_parser.hpp

```diff
@@ -157,9 +165,6 @@ class ResponseParserNg : public NonCopyable, public NonMovable {
         prev_ = cur;
     }
 
-    // XXX parse() and parser_execute() should return Error
-    // and should be tagged with warn_unused_result
-
```

Comment:
Actually, I noticed that exceptions in this case are not that bad, because one gets automatic routing to `handle_error` if an exception occurs during `handle_data`. I should probably write regress for this, tho.